R/expr__string.R

```diff
@@ -232,7 +232,7 @@ ExprStr_len_chars = function() {
 #'
 #' # Series list of strings to Series of concatenated strings
 #' df = pl$DataFrame(list(bar = list(c("a", "b", "c"), c("1", "2", NA))))
-#' df$select(pl$col("bar")$list$eval(pl$col()$str$concat())$list$first())
+#' df$select(pl$col("bar")$list$eval(pl$col("")$str$concat("-"))$list$first())
```

Comment:
Python will raise a warning if the `concat()` argument is missing. I will update that with another PR.

Reply:
See #853